README.md

```diff
@@ -119,26 +119,26 @@ Refer to the [installation guide](docs/installation.md).
 
 The exporter will need to be running in an environment which has access to AWS. The exporter uses the [AWS SDK for Go](https://aws.github.io/aws-sdk-go-v2/docs/getting-started/) and supports providing authentication via [AWS's default credential chain](https://aws.github.io/aws-sdk-go-v2/docs/configuring-sdk/#specifying-credentials). Regardless of the method used to acquire the credentials, some permissions are needed for the exporter to work.
 
-As a quick start, the following IAM policy can be used to grant the permissions for all YACE's features to work. If some of the features are not needed, the policy can be adjusted accordingly, as described in [this section](#required-iam-permissions).
-
+As a quick start, the following IAM policy can be used to grant the all permissions required by YACE 
 ```json
 {
   "Version": "2012-10-17",
   "Statement": [
     {
-      "Sid": "Stmt1674249227793",
       "Action": [
         "tag:GetResources",
         "cloudwatch:GetMetricData",
         "cloudwatch:GetMetricStatistics",
         "cloudwatch:ListMetrics",
-        "ec2:DescribeTags",
-        "ec2:DescribeInstances",
-        "ec2:DescribeRegions",
-        "ec2:DescribeTransitGateway*",
```

Comment:
I'm not quite sure if there are historical reasons for set of ec2 permissions but currently we only call the `DescribeTransitGatewayAttachments` on the ec2 client so this original list seemed unnecessary